app/components/recognition/governing-body-in-time-info.hbs

```diff
@@ -0,0 +1,11 @@
+{{@governingBodyInTime.governingBody.classification.prefLabel}}
```

Comment:
So, here, maybe some modifications would be better. 
As the business has decides that all admin-units can see (and add) recognition for all verenigingen, it would be better to provide more details  about the governing body from WHICH admin unit. 
So the labeling everywhere should be something 
"Burgemeester 2024  Gemeente Antwerpen"  and not only "Burgemeester 2009"

But the trouble is here, you can't use current session for show, only for edit?

Reply:
If the thing with the current session is too complicated, then I think making a searchable list for ALL eenheden is viable alternative. Possibly, this is even better.

Reply:
The linked administrative unit's name is now also show everywhere :+1:  (see updated screenshot in description).